modules/date-picker/main/index.coffee

```diff
@@ -345,20 +379,23 @@ class DatePicker extends hx.EventEmitter
     @options = hx.merge.defined({
       type: 'calendar' # 'calendar' or 'datepicker'
       defaultView: 'm' # 'm' for month, 'y' for year, or 'd' for decade
+      allowViewChange: true # Allow changing between month/year/decade views
       closeOnSelect: true
       selectRange: false
       validRange: undefined
       range: undefined
       showTodayButton: true
       allowInbuiltPicker: true # Option to allow preventing use of the inbuilt datepicker
       disabled: false
+      v2Features: false # Toggle all v2 functionality
```

Comment:
I'm going to split this up and make it an object with individual properties like

```js
v2Flags: {
  modifyDateOnError: true,
  displayLongMonthInCalendar: false,
  setInitialInputValue: true,
  updateVisibleMonthOnDateChange: false,
  allowInputElementContainer: false,
}
```

This allows us to make further v2 changes moving forwards if they are required without breaking the 1.x release for those using the flags to ease migration